atst/domain/authnid/crl/__init__.py

```diff
@@ -170,6 +180,11 @@ def crl_check(self, cert):
             return True
 
         except crypto.X509StoreContextError as err:
+            if err.args[0][0] == CRL_EXPIRED_ERROR_CODE:
+                if app.config.get("CRL_FAIL_OPEN"):
```

Comment:
```suggestion
                if app.config.get("CRL_FAIL_OPEN"):
                    self._log_info(
                        "Encountered expired CRL for certificate with CN {} and issuer CN {}, failing open.".format(
                            parsed.get_subject().CN, parsed.get_issuer().CN
                        )
                    )
```
Trying out the suggestion feature. I think you should see a toggle that allows you to just accept it? Anyway, I think that we should log here when we fail open.